### PR TITLE
Migration of delivery_carrier_label_postlogistics

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

**Comment by yvaucher** [comment]
Travis restarted as #33 is merged

File: delivery_carrier_label_postlogistics/README.rst [hunk 1]
```diff
@@ -95,20 +67,3 @@
 ------------
 
 * Yannick Vaucher <yannick.vaucher@camptocamp.com>
```
Comment (yvaucher) [inline delivery_carrier_label_postlogistics/README.rst:72]: Missing your name here

Plus it miss the OCA footer

File: delivery_carrier_label_postlogistics/README.rst [hunk 2]
```diff
@@ -1,31 +1,3 @@
...
-##############################################################################
-{'name': 'PostLogistics Labels WebService',
- 'version': '1.1',
- 'author': "Camptocamp,Odoo Community Association (OCA)",
- 'maintainer': 'Camptocamp',
- 'category': 'version',
- 'complexity': 'normal',
- 'depends': ['base_delivery_carrier_label'],
- 'description': """
 PostLogistics Labels WebService
```
Comment (yvaucher) [inline delivery_carrier_label_postlogistics/README.rst:4]: Can you add the AGPL-3 tag ? And the default OCA icon as well.

File: delivery_carrier_label_postlogistics/res_config.py [hunk 3]
```diff
@@ -0,0 +1,420 @@
...
+    # def create(self, vals):
+    #     record = super(PostlogisticsConfigSettings, self).create(vals)
+    #     # Hack: to avoid some nasty bug, related fields are not written
+    #     # upon record creation.  Hence we write on those fields here.
+    #     vals = {}
+    #     for fname, field in self._columns.iteritems():
+    #         if isinstance(field, fields.related) and fname in values:
+    #             vals[fname] = values[fname]
+    #     self.write(cr, uid, [rec_id], vals, context)
+    #     return rec_id
```
Comment (yvaucher) [inline delivery_carrier_label_postlogistics/res_config.py]: To remove ?

**Comment by yvaucher** [comment]
Apart for those few OCA visibility remarks looks goods :+1:

**Comment by guewen** [comment]
@yvaucher done

**Comment by yvaucher** [comment]
Thanks, let's merge it

**Pull request merged**